src/include/kdb.h.in

```diff
@@ -134,80 +125,61 @@ int kdbSet (KDB *handle, KeySet *returned,
  **************************************/
 
 /* Basic Methods */
-Key *keyNew (const char *keyname, ...) ELEKTRA_SENTINEL;
-Key *keyVNew (const char *keyname, va_list ap);
-
-Key * keyCopy (Key *dest, const Key *source, elektraCopyFlags flags);
-
-int keyClear (Key *key);
-int keyDel (Key *key);
-
-uint16_t keyIncRef (Key *key);
-uint16_t keyDecRef (Key *key);
-uint16_t keyGetRef (const Key *key);
+ElektraKey * elektraKeyNew (const char * keyname, ...) ELEKTRA_SENTINEL;
+ElektraKey * elektraKeyVNew (const char * keyname, va_list ap);
 
-/* Meta Info */
-int keyRewindMeta (Key *key);
-const Key *keyNextMeta (Key *key);
-const Key *keyCurrentMeta (const Key *key);
+ElektraKey * elektraKeyCopy (ElektraKey * dest, const ElektraKey * source, elektraCopyFlags flags);
 
-int keyCopyMeta (Key *dest, const Key *source, const char *metaName);
-int keyCopyAllMeta (Key *dest, const Key *source);
+int elektraKeyClear (ElektraKey * key);
+int elektraKeyDel (ElektraKey * key);
 
-const Key *keyGetMeta (const Key *key, const char* metaName);
-ssize_t    keySetMeta (Key *key, const char* metaName,
-	const char *newMetaString);
-KeySet * keyMeta (Key * key);
+uint16_t elektraKeyIncRef (ElektraKey * key);
+uint16_t elektraKeyDecRef (ElektraKey * key);
+uint16_t elektraKeyGetRef (const ElektraKey * key);
```

Comment:
```suggestion
```

should be moved to operations lib (see #4245)

Reply:
To be discussed in #4245 where it belongs best. Maybe we need some diagnostic lib.